client/asset/eth/multirpc.go

```diff
@@ -1440,15 +1441,15 @@ func newCompatibilityTests(cb bind.ContractBackend, net dex.Network, log slog.Lo
 		readIt := func(path string) string {
 			b, err := os.ReadFile(path)
 			if err != nil {
-				log.Errorf("Problem reading simnet testing file: %v", err)
+				panic(fmt.Sprintf("Problem reading simnet testing file %q: %v", path, err))
 			}
-			return strings.TrimRight(string(b), "\r\n")
+			return strings.TrimSpace(string(b)) // mainly the trailing "\r\n"
 		}
 		usdc = common.HexToAddress(readIt(tContractFile))
 		txHash = common.HexToHash(readIt(tTxHashFile))
 		blockHash = common.HexToHash(readIt(tBlockHashFile))
-	default:
-		log.Errorf("Unknown net %v in compatibility tests. Testing data not initiated.", net)
+	default: // caller should have checked though
+		panic(fmt.Sprintf("Unknown net %v in compatibility tests. Testing data not initiated.", net))
```

Comment:
In the product code, there are probably 30 places it would panic before getting here anyway.  It's virtually impossible to get an unrecognized `dex.Network` here with the product.